tests/unit/Endpoints/Webhooks/InfoHandlerTest.php

```diff
@@ -2,21 +2,64 @@
 
 namespace StoreKeeper\WooCommerce\B2C\UnitTest\Endpoints\Webhooks;
 
+use Exception;
+use Mockery\MockInterface;
+use StoreKeeper\WooCommerce\B2C\Commands\ProcessAllTasks;
 use StoreKeeper\WooCommerce\B2C\Endpoints\Webhooks\InfoHandler;
+use StoreKeeper\WooCommerce\B2C\Exports\OrderExport;
 use StoreKeeper\WooCommerce\B2C\Models\TaskModel;
-use StoreKeeper\WooCommerce\B2C\Models\WebhookLogModel;
 use StoreKeeper\WooCommerce\B2C\Options\StoreKeeperOptions;
+use StoreKeeper\WooCommerce\B2C\Tools\OrderHandler;
+use StoreKeeper\WooCommerce\B2C\Tools\StoreKeeperApi;
+use StoreKeeper\WooCommerce\B2C\Tools\TaskHandler;
 use StoreKeeper\WooCommerce\B2C\UnitTest\Endpoints\AbstractTest;
+use Throwable;
 
 class InfoHandlerTest extends AbstractTest
 {
+    const DATA_DUMP_FOLDER = 'endpoints/webhooks/infoHandler';
+
     /**
-     * test if info works.
-     *
-     * @throws \Throwable
+     * @throws Throwable
      */
     public function testHandleOk()
     {
+        $this->initApiConnection();
+
+        $this->mockApiCallsFromDirectory(self::DATA_DUMP_FOLDER, false);
+
+        $firstFailedOrderId = $this->createWoocommerceOrder();
```

Comment:
can you add comment why those 2 are expected to fail?

Reply:
Added comment.